course/sitemaps.py

```diff
@@ -18,33 +17,29 @@ def items(self):
     def location(self, item):
         return reverse(item)
 
+    def priority(self, item):
+        return self.base_priority
 
-class InstanceSitemap(sitemaps.Sitemap):
-    priority = 1.0
+
+class InstanceSitemap(AplusSitemap):
     changefreq = 'daily'
 
     def items(self):
         return CourseInstance.objects.filter(
             view_content_to=CourseInstance.VIEW_ACCESS.PUBLIC,
         )
 
-    def location(self, item):
-        return item.get_display_url()
-
 
-class ModuleSitemap(sitemaps.Sitemap):
-    priority = 0.2
+class ModuleSitemap(AplusSitemap):
     changefreq = 'daily'
+    base_priority = 0.1
```

Comment:
These are similar to the course table of contents pages, so they should have the same priority. I thought it could be raised to `base_priority = 0.2`.